### ISSUE TITLE
UX enhancement suggestions

### ISSUE THREAD

**Issue by sfkeller** [wondie/batch_gps_importer#12]
We're not only trusted QGIS plugin programmers here but I'm also a UX engineer by education. So I've following enhancement suggestions which imply also some changes in the plugin logic:

A. Add the plugin not in a sub menu but directly in the menu: Current situation:
![image](https://github.com/user-attachments/assets/77ddcb87-b28e-41e6-8129-2dc39bed8245)
Expected situation: Plugin directly available in the menu "Vector".

B. Then there are some issues in the dialog. Here's a mock design with the current situation first followed by a proposal: 
![image](https://github.com/user-attachments/assets/e8598b3d-b2a2-43cc-89aa-4e60aa95ab46)

C. Instead of dynamic help (which currently doesn't work anyway) I'd suggest to have hover text.

**Comment by wondie** [comment]
@sfkeller, thank you very much for the suggestions! I will add more gaps to the border. With regards to having no sub-folder, I will consider it and see if it is acceptable in all use cases. 

On which QGIS version is the dynamic help not working?

**Comment by sfkeller** [comment]
What's also confusing is, that Geometry type has no default upto now (I'd suggest Point or LineString depending on GPX type) and that it offers "Polygon" which is no existing geometry type in GPX.

**Comment by wondie** [comment]
I agree; I need to add the default geometry type. Probably, line. The Polygon is not a data type of GPX. It is generated by connecting and converting the waypoints or tracks to the polygon.

**Comment by wondie** [comment]
@sfkeller 1.1.0 release added. Most suggestions are incorporated. Checkout the [change log](https://github.com/wondie/batch_gps_importer/releases/tag/v1.1.0).

**Comment by sfkeller** [comment]
Wow. That's great.

Did @SamMeuli already tell you about this open tutorial (https://openschoolmaps.ch/lehrmittel/weitere_qgis-themen/eine_karte_deiner_reise_mit_qgis_erstellen/eine_karte_deiner_reise_mit_qgis_erstellen.html) where we mention the ‘Batch GPS Importer’ plugin (it's in German)? Now Samuel and I have to adapt it to Figure 3 and some text... :-).

**Comment by wondie** [comment]
Great to see a tutorial in German! Thank you for the suggestions and @SamMeuli contirbutions. I didn't know you were working together!
